monday/resources/columns.py

```diff
@@ -0,0 +1,13 @@
+from monday.query_joins import update_multiple_column_values_query, create_column
+
+from monday.resources.base import BaseResource
+
+
+class ColumnsResource(BaseResource):
+    def create_column(self, board_id):
+        query = create_column(board_id)
```

Comment:
`column_title` is also required for this method.

Reply:
Fixed.